incubator/patroni/templates/statefulset-patroni.yaml

```diff
@@ -74,7 +77,7 @@ spec:
   - metadata:
       name: pgdata
       annotations:
-        volume.alpha.kubernetes.io/storage-class: anything
+        volume.alpha.kubernetes.io/storage-class: default
```

Comment:
This should conditionally be a beta annotation if a storage class is given, alpha otherwise.

Now that 1.6 is out, we could also just stick with the beta annotation, or use the storageClassName field, but we still need to roll that change out to all charts.